falkordb/falkordb.py

```
@@ -225,6 +227,49 @@
+            the FalkorDB setup.
+            ValueError: If the `mode` is neither Sentinel nor Cluster.
+        """
+        # decide if it's Sentinel or cluster
+        
+        if hasattr(self, 'sentinel') and self.sentinel is not None:
+            try:
+                replica_hostnames = self.sentinel.discover_slaves(service_name=self.service_name)
+                if len(replica_hostnames) == 0:
+                    raise ConnectionError("Unable to get replica hostname.")
+                return [(host, int(port)) for host, port in replica_hostnames]
+            except redis.RedisError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)